.github/workflows/bionic_build.yml

```diff
@@ -0,0 +1,51 @@
+name: Bionic-Build
+
+on:
+  push:
```

Comment:
Is it really valuable to have CI run on your personal branch every time you push a commit? I've found that I never really check those builds and having `on: pull_request` is sufficient. It may not matter much here, but you can run into issues with private repositories (and possibly at the organization level) where you use up all your allowable CI minutes (with certain types of accounts, namely free accounts) because CI runs every time you push a commit even though you probably didn't intend for it to run

Reply:
Yea, push should be only enabled for the master branch. I have updated this to include branches master, and feature/CommandLanguage.